randovania/data/json_data/prime1.json

```diff
@@ -288,6 +288,27 @@
                 "short_name": "Newborn",
                 "max_capacity": 1,
                 "extra": null
+            },
+            {
+                "index": 47,
```

Comment:
Same as Energy Tank, the correct thing is support a different id.

Though Prime 1 doesn't actually have a "Item Percent" item, but it should be fine to create a fake one with a >1000 id as well.